### PR TITLE
[16.10] BUG: Reload converter tools after toolbox reload

### PR DESCRIPTION
**Pull request by mvdbeek**
The first commit is a testcase demonstrating the problem.
To solve this we just reload the converter tools (from the tool cache).

### CODE REVIEW

File: lib/galaxy/jobs/runners/condor.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def queue_job( self, job_wrapper ):
         container = None
         universe = query_params.get('universe', None)
         if universe and universe.strip().lower() == 'docker':
-            container = self.find_container( job_wrapper )
+            container = self._find_container( job_wrapper )
```
Comment (martenson) [inline lib/galaxy/jobs/runners/condor.py:69]: I think this has already been merged, probably just GH display glitch.

Comment (mvdbeek) [reply lib/galaxy/jobs/runners/condor.py:69]: yeah, I've just changed the base branch.

File: test/api/test_datatypes.py [hunk 2]
```diff
@@ -56,6 +57,19 @@ def test_converters( self ):
 
         assert found_fasta_to_tabular
 
+    def test_converter_present_after_toolbox_reload( self ):
+        response = self._get( "tools", data={'tool_id': 'CONVERTER_wiggle_to_interval_0'} )
```
Comment (jmchilton) [inline test/api/test_datatypes.py]: Can you use ``CONVERTER_fasta_to_tabular`` since it is actually referenced in ``test/functional/tools/sample_datatypes_conf.xml``? I think it should be available under more precise testing setups.

Comment (mvdbeek) [reply test/api/test_datatypes.py]: Sure, no problem.

File: lib/galaxy/webapps/galaxy/api/configuration.py [hunk 3]
```diff
@@ -85,7 +85,7 @@ def tool_lineages(self, trans):
 
     @expose_api
     @require_admin
-    def reload_toolbox(self, trans):
+    def reload_toolbox(self, trans, **kwds):
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/configuration.py:88]: Is this change needed?

Comment (mvdbeek) [reply lib/galaxy/webapps/galaxy/api/configuration.py:88]: I introduced this change to be able to call this from the API -- otherwise it would pass the 'key' param containing the API key to `reload_toolbox`, which would just fail.

File: lib/galaxy/jobs/runners/condor.py (hunk 1 above)
Comment (nsoranzo) [reply lib/galaxy/jobs/runners/condor.py:69]: The commit has a different id, maybe you can rebase your branch on upstream/release_16.10 .

File: lib/galaxy/webapps/galaxy/api/configuration.py (hunk 3 above)
Comment (mvdbeek) [reply lib/galaxy/webapps/galaxy/api/configuration.py:88]: I suppose this wasn't necessary in https://github.com/galaxyproject/galaxy/blob/dev/templates/admin/reload_tool.mako#L18 because javascript was handling the session?!

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by blankenberg** [comment]
Thanks @mvdbeek
